### ISSUE TITLE
[Enhancement] If there are multiple import statements with the same alias, the alias that appears later takes effect.

### ISSUE THREAD

**Issue by zong-zhe** [kcl-lang/kcl#441]
## Enhancement
If there are multiple import statements with the same alias, the alias that appears later takes effect.
```
# The content of 'b/b.k' is "b=1"
import .b.b as b 

# The content of 'b.k' is "b=2"
import .b as b 

# got "c=1", but "c=2" may be better.
c = b.b 
```

And this feature is not described in the https://kcl-lang.io/.

**Comment by Peefy** [comment]
For the case mentioned in the Issue, error reporting is the best choice. In addition, for the KPM tool, it will pull KCL packages from various hubs/repos/registries, such as Git, OCI, ArtifactHub, Url, etc. It needs to have the ability to set the pull package name and import it from the KCL code.

**Comment by Peefy** [comment]
Closed by #727
